catalog/core/catalog-core-camelcomponent/src/test/java/ddf/camel/component/catalog/content/FileSystemPersistenceProviderTest.java

```diff
@@ -56,7 +57,8 @@ public void setup() {
         fileSystemPersistenceProvider = new FileSystemPersistenceProvider("test");
         fileSystemPersistenceProvider.fileSystemDataAccessObject = mockFileSystemDao;
         assertThat(fileSystemPersistenceProvider.getMapStorePath(),
-                is(fileSystemPersistenceProvider.getPersistencePath() + "test" + File.separator));
+                is(Paths.get(fileSystemPersistenceProvider.getPersistencePath(), "test")
```

Comment:
This unit test _should_ be failing. Your implementation of `getMapStorePath` is still tacking on `File.separator` to the end (right or wrong, I'm not clear at the moment); however, here, you're comparing it against a path without the separator character appended.